src/lib/query-helpers.js

```diff
@@ -1,50 +1,35 @@
+import { getCoreComponents } from '../preset/core-components';
```

Comment:
I was a little surprised to see this was coupled to the preset like this. I think I would have expected the configuration of this to be independent of the preset

Reply:
The reason for this was so that we could ensure the components the tests could interact with were mocked by the preset. Do you have an idea how we could decouple it?

Reply:
this doesn't happen anymore, lib doesn't reference the preset directly now.